src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -313,10 +328,70 @@ export function InternalDashboardTopNav({
       });
     }
     if (showWriteControls && managed) {
-      allBadges.push(getManagedContentBadge(dashboardManagedBadge.getTooltip()));
+      const renderCustomBadge = () => {
+        const badgeButton = (
+          // @ts-ignore - EuiBadge href type issue
+          <EuiBadge
+            {...getManagedContentBadge(dashboardManagedBadge.getText(), false)}
+            onClick={() => setIsPopoverOpen(!isPopoverOpen)}
+          >
+            {managedText}
+          </EuiBadge>
+        );
+        return (
+          <EuiPopover
+            button={badgeButton}
+            isOpen={isPopoverOpen}
+            closePopover={() => setIsPopoverOpen(false)}
+            className="eui-hideFor--s eui-hideFor--xs"
+            data-test-subj="managedContentPopover"
+          >
+            <EuiFlexItem>
+              <EuiText
+                size="s"
+                aria-label={text}
+                css={css`
+                  max-width: 300px;
+                `}
+              >
+                {text}
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiButtonEmpty
+                size="s"
+                disabled={isLoading}
+                aria-label={buttonText}
+                onClick={() => {
+                  setIsLoading(true);
+                  dashboard.runInteractiveSave(viewMode);
+                  setIsLoading(false);
```

Comment:
Can you explain why we are keeping this `loading` state even though the click is now just opening the modal? I'm not sure it's necessary